cypress/integration/user_account.spec.js

```diff
@@ -0,0 +1,116 @@
+/// <reference types="cypress" />
+
+describe("Tests for user's profile page", () => {
+    const testUser = {
```

Comment:
the same user is being used in multiple spec files, can we abstract it out and avoid duplicate code?
Suppose user model changes in future, we will have to update all files.

Reply:
I have moved the test data to the `cypress.json` file, and removed the redundant code. 
However, some test data that is unique to a particular test is kept within the test file.